pkg/cmd/clustertask/testdata/Test_ClusterTask_Start-Dry_Run_with_--use-param-defaults_and_specified_params.golden

```diff
@@ -11,9 +11,7 @@ spec:
     - name: myarg
       value: value1
     - name: print
-      value:
-      - boom
-      - boom
+      value: boom,boom
```

Comment:
This is something I feel should not change

Reply:
Yes, updated this test and the `v1beta1` test to use `ParamTypeArray` instead of `ParamTypeString`.